meteor-app/imports/api/orders/schema.js

```diff
@@ -0,0 +1,35 @@
+import { Mongo } from 'meteor/mongo'
+import SimpleSchema from 'simpl-schema'
+import { orderedParts } from '/imports/api/orders/orderedParts'
+import CONSTANTS from '/imports/api/constants.js'
+
+
+import { REGEX_ID, createdAt, updatedAt } from '/imports/api/schema'
+
+const Orders = new Mongo.Collection('orders')
+
+export const OrdersSchema = new SimpleSchema({
+  _id: REGEX_ID,
```

Comment:
This is wrong. If you look at the definition `REGEX_ID` is a regular expression. I am surprised that this works at all, although it's deeper in the code than SimpleSchema, so it's probably ignored. You need to use `RegExId` because that is a full field definition, type=string and with a regex on it.